capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    super.upcase
+  end
```

Comment:
- [ ] This method returns the name in all capital letter, but the requirements says it should capitalize the name; that is, it should have the first letter capitalized. example; `maximilianus` should return `Maximilianus` and not `MAXIMILIANUS`. So please let's use the [capitalize ](https://www.geeksforgeeks.org/ruby-string-capitalize-method/)method in ruby to achieve this.